woocommerce/woocommerce
# Remove shipping modal fonts

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:

Fixes https://github.com/woocommerce/woocommerce/issues/50398





In https://github.com/woocommerce/woocommerce/issues/41955 fonts were reverted for main screens but remained for modals. This PR fully removes the new font from modals as well.

### Before

<img width="637" alt="image" src="https://github.com/user-attachments/assets/5202d52c-4829-4115-aa32-3d0464433f29">

### After

<img width="645" alt="image" src="https://github.com/user-attachments/assets/42e28d8a-46be-4e8c-80d7-06a92f8d36dc">



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. WooCommerce > Settings > Shipping
2. Add a new Shipping Zone and method. When the method modal is open, make sure the font is the default.
3. Do the same for Shipping Classes



### Changelog entry







<details>

<summary>Changelog Entry Details</summary>

#### Significance




#### Type




#### Message 

Make shipping modals the same font as the Settings screens

</details>

<details>

<summary>Changelog Entry Comment</summary>

#### Comment 

</details>
